joomla/joomla-cms
# [4.0] Namespace com_redirect

Pull Request for Issue # .

### Summary of Changes
This simple PR convert com_redirect component to namespace MVC for Joomla 4.

### Testing Instructions
1. Install latest Joomla 4.0-dev with sample data, then apply patch (or better install my branch https://github.com/joomdonation/joomla-cms/archive/namespace_com_redirect.zip )

2. Login to administrator area of the site, access to Components -> Redirect, try to add/edit/enable/disable... links, make sure it is working as expected
